zip-1337.md

```diff
@@ -0,0 +1,156 @@
+
+The crux of this proposal still centers around the idea that the Zcash Foundation and ECC share responsibility for protocol development, and assumes an eventual 2-of-2 agreement on the trademark and that both continue developing consensus-compatible node software that interacts with the Zcash network. But it mandates accountability for disbursement of funds to the ECC/Zcash Foundation, and expands outside stakeholder input on funds that _wouldn't_ be earmarked for the ECC/Zcash Foundation (similar to Placeholder's earlier version of their proposal), while it doesn’t preclude the possibility of migrating to broader “2-of-3” later on.
+
+### Why a trust?
+
+The main reason: reducing complexity creep in consensus code so the ECC and Zcash Foundation can budget for other features in NU4. Rather than try to incorporate some complex mechanism for dev fund disbursements on-chain, we can meet the NU4 deadline with the simplest possible code-change and spend more time ironing out the details of the trust "off-chain." Since both the ECC and the Zcash Foundation are based in the US, using a trust with well-specified criteria for disbursements is a reasonable path. This also fits in nicely with [lex-node's proposal](https://forum.zcashcommunity.com/t/dev-fund-supplemental-proposal-enforce-devfund-commitments-with-legal-charter) for legal covenants on funding.
+
+Security and Privacy Considerations
+===================================
+
+The biggest issue is custody of the funds under the trust's control, but I suspect this can be managed with a partnership with a custody partner. There's also the issue that non-public information would need to be verified and validated by the trust, but I view this as a net positive for the community. ("transparency for organizations, privacy for individuals")
+
+Reference implementation
+========================
+
+TBD, but it should be relatively simple to code in both zebra and zcashd.
+
+Issues and further discussion
+=============================
+
+- What are the tax implications for setting up the trust?
+- Who bears the cost of running the trust? (I think jointly split between the ECC and ZF makes sense)
```

Comment:
I might have missed it, but a question I have is about governance of the trust itself. Who oversees that and makes decisions there?

Reply:
I was hoping for trustees selected by ECC/ZF per above — and hoping that the actual management would be extremely straightforward (since the goal of the trust is to provide specific guidelines / instructions to be acted on). However, I think it may be better for there to be separate addresses for each beneficiary and leave it to each org. I will change the proposal shortly this way too.

Reply:
@acityinohio in that case it will be up to "the community" to make sure ZF and ECC are following through on transparency and governance expectations, and to fork out the rewards if they are not?

Reply:
Yup that's right, which is more of a hard veto I know.

Reply:
...I changed my mind on this :grimacing: In thinking it through I think the trust is still the simplest option for disbursement even with the trustee question.